lua/cfc_entity_stubber/m9k/heavy_weapons/m9k_minigun.lua

```diff
@@ -0,0 +1,21 @@
+
+AddCSLuaFile()
+
+CFC_M9k_Stubber.registerStub( function()
+    local SWEP = weapons.GetStored( "m9k_minigun" )
```

Comment:
```suggestion
    local SWEP = cfcEntityStubber.getWeapon( "m9k_ak47" )
```
Use the dedicated stubber function instead

Reply:
https://github.com/CFC-Servers/cfc_entity_stubber/commit/e42e28b115ac944e7fd1e27bb6936935fef602f5
these are stub examples you can take a look at